Source/CombatExtended/Compatibility/Trenches.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using CombatExtended.CombatExtended.LoggerUtils;
+using UnityEngine;
+using Verse;
+using VFESecurity;
+
+namespace CombatExtended.Compatibility
+{
+    [StaticConstructorOnStartup]
+    public class CETrenches
+    {
+	private static bool vfeInstalled;
+	private const string VFES_ModName = "Vanilla Furniture Expanded - Security";
+	static CETrenches()
+	{
+	    vfeInstalled = ModLister.HasActiveModWithName(VFES_ModName);
+	}
+	public static float GetHeightAdjust(IntVec3 cell, Map map)
+	{
+	    if (cell == null || map == null)
+	    {
+		return 0;
+	    }
+	       
+	    if (vfeInstalled)
```

Comment:
Prefer early return

Reply:
I considered that, but in this case if we decide to handle other mods, each one gets its own ` if (modXInstalled)`.